pyuntl/untldoc.py

```diff
@@ -578,19 +579,17 @@ def generate_rdf_xml(dc_dict):
     Returns in XML serialized format.
     """
     rdf_py = dcdict2rdfpy(dc_dict)
-    return rdf_py.serialize(format='pretty-xml')
+    return rdf_py.serialize(format='pretty-xml', encoding='utf-8')
 
 
 def retrieve_vocab():
     """Get the vocabulary data for the field.
 
     Retrieves and returns the vocabulary. Upon failure, returns None.
     """
-    url = VOCABULARIES_URL.replace('all', 'all-verbose')
     try:
-        vocab_data = urllib.request.urlopen(url).read()
```

Comment:
Didn't catch this before, but running flake8 now, I think with this change we no longer need to import `urllib.request` in this file.